### ISSUE TITLE
Running the Wizard with Bun

### ISSUE THREAD

**Issue by HazAT** [getsentry/sentry-wizard#446]
If I run a local build of the Wizard with Bun 1.0 it works

Mostly - there is an issue with clack where it gets stuck in project selection
suddenly stops taking input
<img width="576" alt="image" src="https://github.com/getsentry/sentry-wizard/assets/363802/87977752-f38b-461d-aedc-5bfba7e051f5">
https://github.com/natemoo-re/clack/issues/159

when I run it with `npx`
`bunx @sentry/wizard@latest`

I get an error 
```
/private/tmp/@sentry/wizard@latest--bunx/node_modules/r2/index.js:4
  global.fetch = require('node-fetch')
                 ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /private/tmp/@sentry/wizard@latest--bunx/node_modules/r2/node_modules/node-fetch/src/index.js from /private/tmp/@sentry/wizard@latest--bunx/node_modules/r2/index.js not supported.
Instead change the require of /private/tmp/@sentry/wizard@latest--bunx/node_modules/r2/node_modules/node-fetch/src/index.js in /private/tmp/@sentry/wizard@latest--bunx/node_modules/r2/index.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/private/tmp/@sentry/wizard@latest--bunx/node_modules/r2/index.js:4:18)
    at Object.<anonymous> (/private/tmp/@sentry/wizard@latest--bunx/node_modules/@sentry/wizard/dist/lib/Steps/OpenSentry.js:62:10)
    at Object.<anonymous> (/private/tmp/@sentry/wizard@latest--bunx/node_modules/@sentry/wizard/dist/lib/Steps/index.js:6:20)
    at Object.<anonymous> (/private/tmp/@sentry/wizard@latest--bunx/node_modules/@sentry/wizard/dist/lib/Setup.js:87:25)
    at Object.<anonymous> (/private/tmp/@sentry/wizard@latest--bunx/node_modules/@sentry/wizard/dist/bin.js:28:15) {
  code: 'ERR_REQUIRE_ESM'
}

Node.js v20.2.0
error: "sentry-wizard" exited with code 1 (SIGHUP)
```

**Comment by Lms24** [comment]
I just tried `bunx @sentry/wizard@latest` and with the sourcemaps wizard flow, setting up the Vite plugin in a Vite+React app. No error message so far 🤔  Does this still happen with `bun@1.0.1`?

**Comment by Lms24** [comment]
@AbhiPrasad good point, opened a separate issue to track.

**Comment by HazAT** [comment]
bad bot

**Comment by Lms24** [comment]
@HazAT is this still an issue? I couldn't reproduce it back then and the `trustedDependencies` task is tracked separately in #476.

**Comment by braydend** [comment]
I'm still having this issue. I was able to run `npx @sentry/wizard@latest -i sveltekit` (Node v21) as a workaround and it seems to have worked

**Comment by Lms24** [comment]
@braydend thanks for letting us know! Did you run the wizard with the latest bun version?

**Comment by braydend** [comment]
> @braydend thanks for letting us know! Did you run the wizard with the latest bun version?

These are the versions I am using:
Bun: 1.0.2
Sveltekit: ^2.0.0
Svelte: 5.0.0-next.33

**Comment by Lms24** [comment]
Can you try updating Bun to the latest version? According to https://github.com/natemoo-re/clack/issues/170#issuecomment-1857101917 there was a bug that might be causing your problems and it was fixed in Bun 1.0.18.

**Comment by HazAT** [comment]
tried it with Bun 1.1.8 and latest version of the wizard - it works now
